test/black-box/RpcFindItems.test.ts

```diff
@@ -0,0 +1,329 @@
+            publicKey: 'publickey1'
+        }
+    };
+
+    const testDataTwo = {
+        hash: 'hash2',
+        itemInformation: {
+            title: 'title UPDATED',
+            shortDescription: 'item UPDATED',
+            longDescription: 'item UPDATED',
+            itemCategory: {
+                name: 'CHILD',
+                description: 'item UPDATED',
+                key: 'CHILD'
+            },
+            itemLocation: {
+                region: Country.FINLAND,
+                address: 'asdf UPDATED',
+                locationMarker: {
+                    markerTitle: 'UPDATED',
+                    markerText: 'UPDATED',
+                    lat: 33.333,
+                    lng: 44.333
+                }
+            },
+            shippingDestinations: [{
+                country: Country.EU,
+                shippingAvailability: ShippingAvailability.SHIPS
+            }],
+            itemImages: [{
+                hash: 'imagehash1 UPDATED',
+                data: {
+                    dataId: 'dataid1 UPDATED',
+                    protocol: ImageDataProtocolType.IPFS,
+                    encoding: null,
+                    data: null
+                }
+            }]
+        },
+        paymentInformation: {
+            type: PaymentType.FREE,
+            escrow: {
+                type: EscrowType.MAD,
+                ratio: {
+                    buyer: 1,
+                    seller: 1
+                }
+            },
+            itemPrice: {
+                currency: Currency.PARTICL,
+                basePrice: 3.333,
+                shippingPrice: {
+                    domestic: 1.111,
+                    international: 2.222
+                },
+                address: {
+                    type: CryptocurrencyAddressType.STEALTH,
+                    address: '1234 UPDATED'
+                }
+            }
+        },
+        messagingInformation: {
+            protocol: MessagingProtocolType.SMSG,
+            publicKey: 'publickey1 UPDATED'
+        }
+    };
+
+    let createdId;
+    let createdHash;
+    let createdHashTwo;
+    let createdCategory;
+    let createdItemInformation;
+    beforeAll(async () => {
+        const command = new DatabaseResetCommand();
+        await command.run();
+    });
+
+
+    test('Should list listing items with our new create one', async () => {
+        const res = await api('POST', '/api/listing-items', {
+            body: testData
+        });
+        res.expectJson();
+        res.expectStatusCode(201);
+        res.expectData(keys);
+        createdId = res.getData()['id'];
+        createdHash = res.getData()['hash'];
+        createdItemInformation = res.getData()['ItemInformation'];
+        createdCategory = createdItemInformation.ItemCategory;
+
+        // create second listing item
+        const resTwo = await api('POST', '/api/listing-items', {
+            body: testDataTwo
+        });
+
```

Comment:
you are inserting the test data, but not actually testing that you can find the items using the rpc command.

Reply:
I think you should actually move these test data inserts to the test below.